src/ResourceManager/EventHub/Commands.EventHub/Commands.EventHub.csproj

```diff
@@ -0,0 +1,192 @@
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Cmdlets\AzureEventHubsCmdletBase.cs" />
+    <Compile Include="Cmdlets\ConsumerGroup\GetAzureEventHubConsumerGroups.cs" />
+    <Compile Include="Cmdlets\ConsumerGroup\NewAzureEventHubConsumerGroups.cs" />
+    <Compile Include="Cmdlets\ConsumerGroup\RemoveAzureEventHubConsumerGroups.cs" />
+    <Compile Include="Cmdlets\ConsumerGroup\SetAzureEventHubConsumerGroups.cs" />
+    <Compile Include="Cmdlets\EventHub\GetAzureEventHubAuthorizationRules.cs" />
+    <Compile Include="Cmdlets\EventHub\GetAzureEventHubKey.cs" />
+    <Compile Include="Cmdlets\EventHub\GetAzureEventHubs.cs" />
+    <Compile Include="Cmdlets\EventHub\NewAzureEventHub.cs" />
+    <Compile Include="Cmdlets\EventHub\NewAzureEventHubAuthorizationRules.cs" />
+    <Compile Include="Cmdlets\EventHub\RemoveAzureEventHub.cs" />
+    <Compile Include="Cmdlets\EventHub\RemoveAzureEventHubAuthorizationRules.cs" />
+    <Compile Include="Cmdlets\EventHub\SetAzureEventHub.cs" />
+    <Compile Include="Cmdlets\EventHub\SetAzureEventHubAuthorizationRules.cs" />
+    <Compile Include="Cmdlets\EventHub\NewAzureEventHubKey.cs" />
+    <Compile Include="Cmdlets\Namespace\GetAzureEventHubNamespace.cs" />
+    <Compile Include="Cmdlets\Namespace\GetAzureEventHubNamespaceAuthorizationRules.cs" />
+    <Compile Include="Cmdlets\Namespace\GetAzureEventHubNamespaceKeys.cs" />
+    <Compile Include="Cmdlets\Namespace\NewAzureEventHubNamespace.cs" />
+    <Compile Include="Cmdlets\Namespace\NewAzureEventHubNamespaceAuthorizationRules.cs" />
+    <Compile Include="Cmdlets\Namespace\RemoveAzureEventHubNamespace.cs" />
+    <Compile Include="Cmdlets\Namespace\RemoveAzureEventHubNamespaceAuthorizationRules.cs" />
+    <Compile Include="Cmdlets\Namespace\SetAzureEventHubNamespace.cs" />
+    <Compile Include="Cmdlets\Namespace\SetAzureEventHubNamespaceAuthorizationRules.cs" />
+    <Compile Include="Cmdlets\Namespace\NewAzureEventHubNameSpaceKey.cs" />
+    <Compile Include="Models\ConsumerGroupAttributes.cs" />
+    <Compile Include="Models\EventHubsAttributes.cs" />
+    <Compile Include="Models\NamespaceAttributes.cs" />
+    <Compile Include="Models\NamespaceState.cs" />
+    <Compile Include="Models\ListKeysAttributes.cs" />
+    <Compile Include="Models\RegenerateKeysAttributes.cs" />
+    <Compile Include="Models\SharedAccessAuthorizationRuleAttributes.cs" />
+    <Compile Include="Resources.Designer.cs" />
+    <Compile Include="Utilities\EventHubsClient.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="AzureRM.EventHub.psd1">
```

Comment:
You need to copy ..\AzureRM.EventHub.psd1, as the one in this directory should be removed.